### PR TITLE
zh: update contents about relay

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- mark `relay log` is not supported yet (two many people have known it, so they may become confused if we remove contents directly)
- update some contents relative to it

NOTE: we'll add it back later.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GMHDBJD @ireneontheway PTAL

File: zh/usage-scenario-master-slave-switch.md [hunk 1]
```diff
@@ -27,29 +27,27 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-master-slave-switch/'
...
 4. 确保满足以下关系，否则不支持将 DM-worker 连接切换到相应的 MySQL 实例：
-    - `gtid-W` 包含 `gtid-P`（`gtid-P` 可以为空）
-    - `gtid-E` 包含 `gtid-W`
+    - `gtid-S` 包含 `gtid-P`（`gtid-P` 可以为空）
+    - `gtid-E` 包含 `gtid-S`
 5. 使用 `stop-task` 命令停止所有运行中的数据迁移任务。
-6. 更新 `inventory.ini` 中的 DM-worker 配置，并使用 DM-Ansible 对 DM-worker 进行滚动升级操作。
-7. 使用 `start-task` 命令重新启动数据迁移任务。
+6. 使用 `operate-source stop` 命令从 DM 集群中移除原 MySQL 实例地址对应的 source 配置。
+7. 更新 source 配置文件中的 MySQL 实例地址并使用 `operate-source create` 将新的 source 配置重新加载到 DM 集群中
```
Comment (GMHDBJD) [inline zh/usage-scenario-master-slave-switch.md:52]:
```suggestion
7. 更新 source 配置文件中的 MySQL 实例地址并使用 `operate-source create` 将新的 source 配置重新加载到 DM 集群中。
```

**Comment by csuzhangxc** [comment]
@GMHDBJD PTAL again

**Comment by csuzhangxc** [comment]
@TomShawn two LGTMs now, PTAL

File: zh/usage-scenario-master-slave-switch.md [hunk 2]
```diff
@@ -27,29 +27,27 @@ aliases: ['/docs-cn/tidb-data-migration/dev/usage-scenario-master-slave-switch/'
...
 4. 确保满足以下关系，否则不支持将 DM-worker 连接切换到相应的 MySQL 实例：
-    - `gtid-W` 包含 `gtid-P`（`gtid-P` 可以为空）
-    - `gtid-E` 包含 `gtid-W`
+    - `gtid-S` 包含 `gtid-P`（`gtid-P` 可以为空）
+    - `gtid-E` 包含 `gtid-S`
 5. 使用 `stop-task` 命令停止所有运行中的数据迁移任务。
-6. 更新 `inventory.ini` 中的 DM-worker 配置，并使用 DM-Ansible 对 DM-worker 进行滚动升级操作。
-7. 使用 `start-task` 命令重新启动数据迁移任务。
+6. 使用 `operate-source stop` 命令从 DM 集群中移除原 MySQL 实例地址对应的 source 配置。
+7. 更新 source 配置文件中的 MySQL 实例地址并使用 `operate-source create` 将新的 source 配置重新加载到 DM 集群中。
```
Comment (TomShawn) [inline zh/usage-scenario-master-slave-switch.md:52]:
```suggestion
7. 更新 source 配置文件中的 MySQL 实例地址并使用 `operate-source create` 命令将新的 source 配置重新加载到 DM 集群中。
```

**Pull request merged**